### PR TITLE
test: refactor volume test

### PR DESCRIPTION
**Pull request by ZYecho**
Signed-off-by: zhangyue <zy675793960@yeah.net>

### Ⅰ. Describe what this PR did
Add a util func to refactor volume test to make test code more readable

### Ⅱ. Does this pull request fix one issue?

None.

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
None.

### Ⅳ. Describe how to verify it
`go test -gocheck.f PouchVolumeSuite`

### Ⅴ. Special notes for reviews
None.

### CODE REVIEW

**Comment by ZYecho** [comment]
```
Build-agent version 0.1.1073-1f69f340 (2018-11-20T18:07:03+0000)
Starting container pouchcontainer/pouchlinter:v0.1.2
  image cache not found on this host, downloading pouchcontainer/pouchlinter:v0.1.2
```

**Comment by allencloud** [comment]
I have rerun the linter CI. The linter CI passes, and while the travisCI fails:
```
----------------------------------------------------------------------
FAIL: /home/travis/gopath/src/github.com/alibaba/pouch/test/cli_rmi_test.go:68: PouchRmiSuite.TestRmiByImageDigestID
/home/travis/gopath/src/github.com/alibaba/pouch/test/cli_rmi_test.go:69:
    command.PouchRun("pull", helloworldImage).Assert(c, icmd.Success)
/home/travis/gopath/src/github.com/alibaba/pouch/vendor/github.com/gotestyourself/gotestyourself/icmd/command.go:61:
    t.Fatalf("at %s:%d - %s\n", filepath.Base(file), line, err.Error())
... Error: at cli_rmi_test.go:69 - 
Command:  /usr/local/bin/pouch pull registry.hub.docker.com/library/hello-world:linux
ExitCode: 1
Error:    exit status 1
Stdout:   registry.hub.docker.com/library/hello-world:linux: resolving
Stderr:   Error: failed to display progress: failed to pull image: registry.hub.docker.com/library/hello-world:linux not found
Failures:
ExitCode was 1 expected 0
Expected no error
----------------------------------------------------------------------
```

I will re-run travis CI to check if this is a flaky test. @ZYecho

File: test/cli_volume_test.go [hunk 1]
```diff
@@ -305,49 +299,64 @@ func (suite *PouchVolumeSuite) TestVolumeListOptions(c *check.C) {
...
-		if strings.Contains(line, volumeName) {
-			if !strings.Contains(line, "local") ||
-				!strings.Contains(line, DefaultVolumeMountPath) {
-				c.Errorf("list result have no driver or name or size or mountpoint, line: %s", line)
-				break
-			}
+	lines := volumesToKV(ret.Stdout())
+	for _, line := range lines {
+		c.Assert(line[0], check.Equals, "local")
+		if !strings.Contains(line[3], DefaultVolumeMountPath) {
```
Comment (allencloud) [inline test/cli_volume_test.go]: I think when you were refactoring this part, you thought of add more checking rules which is described in https://github.com/alibaba/pouch/issues/2497 😄 @ZYecho

Comment (ZYecho) [reply test/cli_volume_test.go]: yep, `check.Contains` is really helpful.

**Comment by allencloud** [comment]
Never mind the linkcheck failure: 
```
FILE: ./docs/features/pouch_with_criu.md
[✖] https://criu.org/Main_Page
[✖] https://criu.org/Linux_kernel
[✖] https://criu.org/Packages

ERROR: dead links found!
```

**Comment by allencloud** [comment]
LGTM

**Pull request merged**